src/patch_parse.c

```diff
@@ -607,6 +611,9 @@ static int parse_hunk_body(
 		line->content_len = ctx->parse_ctx.line_len - prefix;
 		line->content_offset = ctx->parse_ctx.content_len - ctx->parse_ctx.remain_len;
 		line->origin = origin;
+		line->num_lines = 1;
```

Comment:
I'm suspicious of this : is there actually a way of building a text-based diff with embedded newlines 😧 ?

Reply:
I don't think that's a thing.  I don't know how this could ever be anything but 1.

Reply:
Looks like the xdiff code also doesn't set it. The only setters are in diff_print.c.